inc/vcf/fixer.hpp

```diff
@@ -54,13 +56,30 @@ namespace ebi
         virtual void visit(FileformatError &error) override;
         virtual void visit(ChromosomeBodyError &error) override;
         virtual void visit(PositionBodyError &error) override;
+
+        /**
+         * fix: 
+         * - remove duplicate IDs, keep the first one and remove consequent ones
+         */
         virtual void visit(IdBodyError &error) override;
         virtual void visit(ReferenceAlleleBodyError &error) override;
         virtual void visit(AlternateAllelesBodyError &error) override;
         virtual void visit(QualityBodyError &error) override;
+
+        /**
+         * fix:
+         * - if any FILTER string is 0, remove it
+         * - remove duplicate FILTER strings - keep the first and remove consequent ones
+         */
         virtual void visit(FilterBodyError &error) override;
+
         /**
-         * fix: remove the info field that caused the error
+         * fix:
+         * - fix duplicates in INFO field :
+         *     - if even a single value differs, remove all the duplicate key fields
+         *     - else if the duplicate key has the same value, keep the first one and remove all the others
+         * - if the error is recoverable, i.e., it expected an exact value, then replace the error causing INFO field with the correct value
+         * - if the error is irrecoverable, remove the INFO field that caused the error
```

Comment:
Just to be clear on what I meant by capitalization, I meant in plain English. This docstring would look like:

```
/**
 * Fixes duplicates in INFO field.
 * - If even a single value differs, remove all the duplicate key fields
 * - Else if all the duplicate keys have the same value, keep only the first occurrence
 *
 * If the error is recoverable, i.e. it expected an exact value, then replace the error-causing
 * INFO field with the correct value. If the error is irrecoverable, remove the INFO field that 
 * caused the error.
 */
```